walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -20,25 +20,27 @@
 import java.util.Arrays;
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    
-    //initialize availibility collection
     Collection<TimeRange> availibility;
-    
-    
+  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+
     //no attendees -- first test
-    if (events == null){
+    if (request.getAttendees() == null){
         availibility = Arrays.asList(TimeRange.WHOLE_DAY);
     }
 
     //no options too long of a request (how do I access time range here? THis is an error) -- second test
-    else if (request.duration > 24){
-        //availibility = Arrays.asList();
+    else if (request.getDuration() > TimeRange.WHOLE_DAY.duration()){
```

Comment:
same with second test ... is hypothetically returning empty when the duration is longer than a day

Reply:
Same as the above comment